### ISSUE TITLE
Final Report Peer Review - cl2567

### ISSUE THREAD

**Issue by cl3587** [RyChen-Cornell/ORIE4741-Group-Project#12]
This project is aimed to predict the future stock trend based on the quarterly financial statement. Specifically, the project formularized the problem as a classification problem to classify whether the stock price will go up or down based on the patterns of those financial features.

Some techniques they used:

In term of dealing with missing values, they removed companies which contain a lot of missing return values; Also, they tested the correlation among features and normalized the features using Standard Scaler. They also used one-hot encoding to generate new features.

In term of model implementation, they tried different models including SVM; random forest; logistic regression; Neural Network; AdaBoost and XGBoost. The accuracy is mostly around 60% for all the models. This indicates that the boundary of the models may be this. I really like the process of how they reflect on their models' performance. 

Also, there do exist some problems: 
1. They should talk about how they choose different parameters for each model; Tuning the parameters is usually very important for some parameter-sensitive algorithms like Random Forest and NN; 
2. They could calculated the fairness metrics like demographic parity and equalized odds in order to draw a more reliable conclusion of the model fairness;
3. Some non-linear models like NN and AdaBoost/XGBosst need more explanation. Should talk about the motivation of using these complex model since they did not really overperform the linear model significantly.
